content/20.treatments.md

```diff
@@ -897,6 +897,24 @@ They may also engage the host immune response, encouraging the immune system to
 Given the cytokine storm that results from immune system activation in response to the virus, which has been implicated in worsening of the disease, a neutralizing antibody (nAb) may be preferable.
 Upcoming work may explore the specificity of nAbs for their target, mechanisms by which the nAbs impede the virus, and improvements to antibody structure that may enhance the ability of the antibody to block viral activity.
 
+There is also some research into possible potential therapeutics or prophylactics which interact with components of the innate immune response.
+For example, there are a variety of toll-like receptors (TLRs) which are examples of pattern recognition receptors (PRRs), innate immune response components which recognize pathogen-associated molecular patterns (PAMPs) and damage-associated molecular patterns (DAMPs).
+Note that TLRs form a part of innate immune recognition and can more generally contribute to promoting both innate and adaptive responses [@ISBN: 9780815332183].
+In mouse models, poly(I:C) and lipopolysaccharide, which are agonists of toll-like receptors TLR3 and TLR4, respectively, showed protective effects when administered prior to SARS-CoV exposure [@doi:10.1128/JVI.01410-12].
+Therefore, TLR agonists and antagonists hold some potential for broad-spectrum prophylaxis.
+Another type of approach which is being investigated is the possible use of what is termed trained immunity, in particular as elicited by non-SARS-CoV-2 whole-microorganism vaccines (or other microbial stimuli), as a tool which might be shown to generate heterologous protective effects with respect to SARS-CoV-2 susceptibility or severity [@doi:10.1016/j.cell.2020.04.042].
+In a recent review [@doi:10.1038/s41577-020-0285-6], trained immunity was defined as forms of memory which are temporary (e.g., months or years, and reversible), displayed by innate immune cells and innate immune features of other cells, displayed as increased responsiveness to future same or heterologous pathogen infection or sometimes decreased responsiveness or immunological tolerance, and established through epigenetic and metabolic mechanisms.
+One type of stimulus which research indicates can induce trained immunity is bacillus Calmette-Guerin (BCG) vaccination.
+BCG is an attenuated form of bacteria _Mycobacterium bovis_.
+The vaccine is most commonly administered for the prevention of tuberculosis in humans.
+With respect to SARS-CoV-2 specifically, clinical trials in non-SARS-CoV-2-infected adults have been setup to assess the possible efficacy of BCG vaccination (e.g., to assess if it may be effective as a potential prophylactic or potential partial prophylactic in (1) reducing susceptibility / preventing infection and (2) reducing disease severity).
```

Comment:
@cgreene caught that we don't have these clinical trials identifiers cited anywhere -- it doesn't matter whether it's here in this sentence or later in the text, but we definitely need to reference them somewhere

Reply:
I incorporated the NCT# references in the latest file & also (again) with the article reference because it describes some of those trials plus some other trials.